circle.yml

```diff
@@ -1,57 +0,0 @@
-general:
-  branches:
-    ignore:
-      - gh-pages
-  artifacts:
-    - "artifacts/"
-
-machine:
-  node:
-    version: 6
-
-dependencies:
-  cache_directories:
-    - "~/.cache/yarn"
-
-  override:
-    - which node
-
-    # install dependencies
-    - ./scripts/bootstrap-env-ubuntu.sh
-
-    - yarn install
-
-test:
-  override:
-    - node -v
-    - >
-      if [ "$CIRCLE_BRANCH" == 'master' ]; then
-        ./scripts/set-dev-version.js
-      fi;
-    - yarn lint
-    # Limit maxWorkers to 3 to avoid OOM on CircleCI
-    - yarn test-ci -- -- --maxWorkers 3
-    - yarn check-lockfile
-    - yarn run build-dist
-    - yarn run build-deb
-
-    # Test that installing as root works and that it also works
-    # behind a user namespace which Circle CI tests are run under
-    - sudo env "PATH=$PATH" bin/yarn install --force
-
-deployment:
-  release:
-    tag: /v[0-9]+(\.[0-9]+)*/
-    owner: yarnpkg
-    commands:
-      # Only NPM is handled here - All other release files are handled in a webhook.
```

Comment:
Keep this comment in the new version please

Reply:
same as above